odoo/tools/js_transpiler.py

```diff
@@ -251,11 +251,11 @@ def repl(matchobj):
 
 EXPORT_FROM_RE = re.compile(r"""
     ^
-    (?P<space>\s*)                      # space and empty line
-    export\s*                           # export
-    (?P<object>{(\s*\w+\s*,?\s*)*})\s*  # { a, b, c as x, ... }
-    from\s*                             # from
-    (?P<path>(\".*\")|('.*')|(`.*`))    # "file path" ("some/path.js")
+    (?P<space>\s*)                                  # space and empty line
+    export\s*                                       # export
+    (?P<object>{(\s*\w+\s*,?\s*)*})\s*              # { a, b, c as x, ... }
+    from\s*                                         # from
```

Comment:
These lines should not change, same for the rest.

Reply:
It is annoying for sure but
1) We really want to keep the code nice as this file is in its first iterations (so we want a good indentation)
2) Technically, this whole thing is one regex, so the diff is not that shocking
3) fge and I wrote this feature, so no blame get lost
With that, we believe it makes sense to keep it

Reply:
Move the comments to be above the lines they're commenting on then. This style of commenting is horrible for exactly the reason being demonstrated here.